docs/development/tools/subkey.md

```diff
@@ -20,10 +20,10 @@ $ cargo install --force --git https://github.com/paritytech/substrate subkey
 If you already have the [Substrate repository](https://github.com/paritytech/substrate), you can build Subkey with:
 
 ```bash
-$ cargo build -p subkey
+$ cargo build -p subkey --release
 ```
 
-This will install `subkey` in `./target/debug/subkey`.
+This will install `subkey` in `./target/release/subkey`.
```

Comment:
I'm not opposed to this change, but I'm just curious why you decided to recommend release mode here?